### DISCUSSION TITLE
Multi-language support for Github Web

### DISCUSSION THREAD

**Discussion by CodeZhangBorui** [community/community#58591] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

Github has become an international software, which makes Github's multilingual capabilities increasingly important.

Microsoft has produced international language support for Github mobile software, so it is necessary to migrate it to the web side.

So, when will Github web have international language support?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @CodeZhangBorui ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
